### DISCUSSION TITLE
pgai extension for postgres

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [supabase/supabase#29550] [Feature Requests]
[pgai](https://github.com/timescale/pgai) is a postgres extension which makes it possible to invoke various AI workflows within postgres itself: create vector embeddings, perform chat completion, classification, summarization, data enrichment etc.

It would be fantastic if this was available within supabase.

**Comment by Torhamilton** [comment]
Makes a whole lot of sense.

**Reply by vojtatranta** [reply]
This would be a time saver! Please add it!

**Reply by vermilion99red** [reply]
this would be amazing

**Comment by nicolas-geysse** [comment]
Bump !

**Comment by Albertbol** [comment]
Bump

**Comment by ivShoto** [comment]
Bump!

**Comment by christian-at-sevenlab** [comment]
Bump 🤜

**Comment by tobalo** [comment]
Bump ^_^

**Comment by danihenrique** [comment]
Bump!

**Comment by osaidz** [comment]
Bump

**Comment by sleakyo** [comment]
Bumpppp

**Comment by guyettenathanj** [comment]
Looking to gauge demand. Would anyone be willing to pay $50 a month for an extension that did this?

**Comment by oliverbj** [comment]
Bump
